.golangci.yml

```
@@ -19,14 +19,19 @@ linters:
     - ireturn  # ireturn is disabled, since it's not needed.
 
   exclusions:
+    generated: lax
     rules:
       - path: cmd/generator/
         linters:
-        - forbidigo        # fmt functions are not forbidden here
-        - gochecknoglobals # global variables are not forbidden here
+          - forbidigo        # fmt functions are not forbidden here
+          - gochecknoglobals # global variables are not forbidden here
+      - path: cmd/gentypes/
+        linters:
+          - forbidigo        # fmt functions are not forbidden here
+          - gochecknoglobals # global variables are not forbidden here
```

Comments should end with a dot.
```suggestion
          - forbidigo        # fmt functions are not forbidden here.
          - gochecknoglobals # global variables are not forbidden here.
      - path: cmd/gentypes/
        linters:
          - forbidigo        # fmt functions are not forbidden here.
          - gochecknoglobals # global variables are not forbidden here.
```